src/screens/Ticketing/Tickets/ReceiptModal.tsx

```diff
@@ -1,11 +1,11 @@
 import React, {useState} from 'react';
 import {View, TouchableOpacity, Modal, TextInput} from 'react-native';
-import {FareContract, sendReceipt} from '../../../api/fareContracts';
+import {FareContractTicket, sendReceipt} from '../../../api/fareContracts';
 import {StyleSheet} from '../../../theme';
 import Text from '../../../components/text';
 
 interface ModalProps {
-  fareContract?: FareContract;
+  fareContract?: FareContractTicket;
```

Comment:
Note to self: rename this to fareContractTicket

Reply:
I'm not sure if we should go for that naming standard. Unless I've missed something, I think `FareContract` contains tickets, so `FareContractTicket` might be misleading.

Reply:
Aight, reverting these changes. The change was made for better distinction between ` FareContract(Ticket)` and `FareContractType`, but now when the latter is renamed to ` PreassignedFareProduct` there is less need for better distinction.